catalog/dags/elasticsearch_cluster/healthcheck_dag.py

```diff
@@ -24,12 +24,12 @@
 from airflow.exceptions import AirflowSkipException
 from airflow.providers.elasticsearch.hooks.elasticsearch import ElasticsearchPythonHook
 from elasticsearch import Elasticsearch
+from legacy_data_refresh.data_refresh_types import DATA_REFRESH_CONFIGS
```

Comment:
I believe we'll need to add the new folder to the list of first party modules in our isort/rust config so they still show up in the group below.